streamz/dataframe/core.py

```diff
@@ -266,10 +273,10 @@ class DataFrame(Frame, _DataFrameMixin):
     --------
     Series
     """
-    _subtype = pd.DataFrame
+    _subtype = object
```

Comment:
If we're going define this in `__init__` then we can probably remove this from the class definition entirely.

Reply:
Yeah, this seems redundant. I wasn't sure if this variable is referenced before initialization.